modules/standard/IO.chpl

```diff
@@ -5177,6 +5177,62 @@ proc _channel.writebits(v:integral, nbits:integral) throws {
   try this.write(new ioBits(v:uint(64), nbits:int(8)));
 }
 
+/*
+   Write ``numBytes`` of data from a :class:`~CTypes.c_ptr` to a fileWriter
```

Comment:
`fileWriter` here and in the documentation of the other overload should probably be in italics or linked to a class